pythia/modules/fusions.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+"""
+The fusions module contains various Fusions techniques some based on BLOCK:
+Bilinear Superdiagonal Fusion for VQA and VRD. For e.g. LinearSum, ConcatMLP
+taken from https://github.com/Cadene/block.bootstrap.pytorch#fusions.
+
+For implementing your own fusion technique, you need to follow these steps:
+
+.. code::
+    from pythia.common.registry import registry
+    from pythia.modules.fusions import BaseFusion
+    from torch import nn
+
+    @regitery.register_fusion("custom")
+    class CustomFusion(nn.Module):
+        def __init__(self, params=None):
+            super().__init__("Custom")
+
+Then in your model's config you can specify ``fusion`` attribute as follows:
+
+.. code::
```

Comment:
I don't this will automatically create a fusion layer inside the model as there is no fusion base class. So, let's just remove this line 19-27 for now.